compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -143,7 +143,6 @@ declare_lint_pass! {
         UNUSED_VARIABLES,
         USELESS_DEPRECATED,
         WARNINGS,
-        WASM_C_ABI,
```

Comment:
FYI whenever removing something from this list, it must be added to the removed lint list. There is a comment above the list explaining that.
```
//! When removing a lint, make sure to also add a call to `register_removed` in
//! compiler/rustc_lint/src/lib.rs.
```